app/controllers/documents_controller.rb

```diff
@@ -167,4 +168,12 @@ def set_return_to_session
   def return_to_session
     session.delete(:return_to)
   end
+
+  def show_sidebar
+    @show_sidebar = if @planning_application.pre_application? &&
+        Rails.configuration.use_new_sidebar_layout &&
+        !BLOCKED_SIDEBAR_EMAILS.include?(current_user&.email)
+      @planning_application.case_record.tasks.find_by(section: "Validation")
+    end
+  end
```

Comment:
Maybe we could move this up to the `application_controller`?

(It might have to be `show_assessment_sidebar`, `show_validation_sidebar`, etc, to avoid conflicting.)

Reply:
Yes, this is one of the improvements I want to do before making it ready for review. 😄